frontend/src/index.jade

```diff
@@ -85,10 +85,8 @@ html
                 option(value="name dsc", v-if="filterGroupSelection !== 'groupByRepos'") &darr; name
                 option(value="searchPath", v-if="filterGroupSelection !== 'groupByAuthors'") &uarr; repo/branch
                 option(value="searchPath dsc", v-if="filterGroupSelection !== 'groupByAuthors'") &darr; repo/branch
-                option(value="totalCommits", v-if="filterGroupSelection === 'groupByNone'") &uarr; contribution
-                option(value="totalCommits dsc", v-if="filterGroupSelection === 'groupByNone'") &darr; contribution
-                option(value="variance", v-if="filterGroupSelection === 'groupByNone'") &uarr; variance
-                option(value="variance dsc", v-if="filterGroupSelection === 'groupByNone'") &darr; variance
+                option(v-for="option in contributionSelection", v-bind:value="option.value") {{ option.text }}
+                option(v-for="option in varianceSelection", v-bind:value="option.value") {{ option.text }}
```

Comment:
No, what I mean was, why do you need to complex things and have different values shown (according to grouping) in the first place when its already categories under `sort within group`?

Reply:
Do you mean that there are options to sort via contributions and variances in both `sort within group` and `sort group` ?
Contribution in `sort group` is to get all the authors/repos in the ascending order of contribution made. Same for variance.
Isn't that the whole point of this PR? :)

Reply:
Forgot to mention that the contribution and variance values are now the total contribution/variances summed for each author/repo

Reply:
As in, there is no need to emphasize `author variance` vs `repo variance` when its all categories under the `sort groups by`; its already implied. It can all be named as `variance` to avoid complicating the code and confuse the user.

Less complexity also helps to reduce damage and chance of bugs occurs. E.g. occurrence of bug where group by shows `Author` and sort groups by shows `repo contribution`. It may confuse the users and they may not realize that it is a bug.

Reply:
Okay, understood.